widgetssdk/src/main/java/com/glia/widgets/view/VisitorCodeView.kt

```diff
@@ -68,60 +97,43 @@ class VisitorCodeView internal constructor(
         }
     }
 
-    override fun setController(controller: VisitorCodeContract.Controller?) {
-        this.controller = controller
-        this.controller?.setView(this)
+    override fun setController(controller: VisitorCodeContract.Controller) {
+        this.controller = controller!!
```

Comment:
I would try to avoid force unwrapping `!!` as much as possible. So, I would leave `this.controller?.setView(this)`. Is it possible here?

Reply:
Sry, forgot I changed controller to be lateinit, no need for the force wrapping any longer